test/Eq.ts

```diff
@@ -0,0 +1,81 @@
+import * as assert from 'assert'
+import * as E from '../src/Eq'
+import { Eq } from 'fp-ts/lib/Eq'
+
+describe('Eq', () => {
+  it('literal', () => {
+    const eq = E.eq.literal('a', null)
+    assert.deepStrictEqual(eq.equals('a', 'a'), true)
+    assert.deepStrictEqual(eq.equals(null, null), true)
+    assert.deepStrictEqual(eq.equals('a', null), false)
+  })
+
+  it('UnknownArray', () => {
+    const eq = E.UnknownArray
+    assert.deepStrictEqual(eq.equals(['a'], ['a']), true)
+    assert.deepStrictEqual(eq.equals(['a'], ['b']), true)
+    assert.deepStrictEqual(eq.equals(['a'], ['a', 'b']), false)
+  })
+
+  it('UnknownRecord', () => {
+    const eq = E.UnknownRecord
+    assert.deepStrictEqual(eq.equals({}, {}), true)
+    assert.deepStrictEqual(eq.equals({ a: 1 }, { a: 1 }), true)
```

Comment:
From the implementation, it looks like only keys are looked at, not values. If that's right could there be a test here to make that explicit/deliberate?
```suggestion
    assert.deepStrictEqual(eq.equals({ a: 1 }, { a: 1 }), true)
    assert.deepStrictEqual(eq.equals({ a: 1 }, { a: 2 }), true)
```

On that note, it might be good to add something to the docs re why it works that way, since it looks counterintuitive/different from a generic `isEqual` function for arbitrary dictionaries e.g. lodash.isEqual.